task4/task4.ipynb

```diff
@@ -112,41 +131,17 @@
     "            retention_rates_all_cohorts[j][i] = np.NaN\n",
     "    return retention_rates_all_cohorts\n",
     "\n",
-    "get_all_retention_rates_months_after_acquisition = get_all_retention_rates_time_normalized(data)\n",
+    "retention_rates_cohort0 = np.array(get_all_retention_rates_for_cohort(data,0))\n",
     "\n",
-    "avg_monthly_retention_rate_across_cohorts = np.apply_along_axis(np.nanmean,axis = 0, arr = get_all_retention_rates_months_after_acquisition)\n",
+    "active_probs_changing_retention_cohort0 = retention_rates_cohort0\n",
```

Comment:
Using the changing retention rates for cohort 0 instead of average changing rates as the task asks for.